packages/twenty-server/src/engine/metadata-modules/object-metadata/object-metadata.service.ts

```diff
@@ -1260,4 +1279,24 @@ export class ObjectMetadataService extends TypeOrmQueryService<ObjectMetadataEnt
       );
     }
   }
+
+  private async createViewWorkspaceFavorite(
+    workspaceId: string,
+    viewId: string,
+  ): Promise<FavoriteWorkspaceEntity> {
+    const favoriteRepository =
+      await this.twentyORMGlobalManager.getRepositoryForWorkspace<FavoriteWorkspaceEntity>(
+        workspaceId,
+        'favorite',
+      );
+
+    const favoriteCount = await favoriteRepository.count();
+
+    return favoriteRepository.save(
```

Comment:
I don't get this one sorry. We should not have duplicated record since we are currently creating the object and the view. So should I change somthing?